### PR TITLE
Reformatted Ranking samples to width 85

### PR DESCRIPTION
**Pull request by sayanshaw24**
Guidelines followed:

- 85 characters per line
- Use 4 spaces for indentation
- Dot, open parentheses, and function/variable name on same line
- Math operations on same line
- Don't indent comments
- Don't break links
- Don't break a comment if it represents a print output
- Add an extra line after a break if it does not already exist
- Break before "$"

Fix for Issue #3478

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs [hunk 1]
```diff
@@ -8,19 +8,22 @@ namespace Samples.Dynamic.Trainers.Ranking
 {
     public static class FastTree
     {
-        // This example requires installation of additional NuGet package
-        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
+        // This example requires installation of additional NuGet package for 
+        // Microsoft.ML.FastTree at
+		// https://www.nuget.org/packages/Microsoft.ML.FastTree/
```
Comment (natke) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs]: Extra white space

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs [hunk 2]
```diff
@@ -8,19 +8,22 @@ namespace Samples.Dynamic.Trainers.Ranking
...
+			// and as the source of randomness. Setting the seed to a fixed number
+			// in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
             // Create a list of training data points.
             var dataPoints = GenerateRandomDataPoints(1000);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+			// consumable by ML.NET API.
```
Comment (natke) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs]: Extra whitespace

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs [hunk 3]
```diff
@@ -27,39 +32,44 @@ public static void Example()
...
             foreach (var p in predictions)
                 Console.WriteLine($"Label: {p.Label}, Score: {p.Score}");
 
             // Expected output:
-            //   Label: 5, Score: 2.195333
-            //   Label: 4, Score: 0.2596574
-            //   Label: 4, Score: -2.168355
-            //   Label: 1, Score: -3.074823
-            //   Label: 1, Score: -1.523607
+            //   Label: 5, Score: 2.493263
```
Comment (natke) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs]: Did these numbers change for the sample?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs (hunk 1 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs]: Fixed in new commit

Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs]: Fixed in new commit.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs (hunk 2 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/FastTree.cs]: Fixed in new commit.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs (hunk 3 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs]: Yeah they must have changed from a more recent pull.

Comment (natke) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs]: @codemz Is that right? Isn't the same data / same seed / same algorithm?

Comment (natke) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs]: Ok, I see that the sample is using random data points. Thanks

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/PermutationFeatureImportance.cs [hunk 4]
```diff
@@ -78,14 +88,25 @@ private class Data
...
+        /// 
         /// <param name="nExamples">The number of examples.</param>
-        /// <param name="bias">The bias, or offset, in the calculation of the label.</param>
-        /// <param name="weight1">The weight to multiply the first feature with to compute the label.</param>
-        /// <param name="weight2">The weight to multiply the second feature with to compute the label.</param>
-        /// <param name="seed">The seed for generating feature values and label noise.</param>
+        /// 
+        /// <param name="bias">The bias, or offset, in the calculation of the label.
+        /// </param>
+        ///
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/PermutationFeatureImportance.cs]: Empty lines floating around.

Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/PermutationFeatureImportance.cs]: I think for lines 93-105, the empty lines were there originally. Would you like me to get rid of them?

**Pull request merged**